[2024-03-04T17:48:47.114+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.file_sensor_task manual__2024-03-04T17:48:44.892614+00:00 [queued]>
[2024-03-04T17:48:47.120+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gera_dados_turbina.file_sensor_task manual__2024-03-04T17:48:44.892614+00:00 [queued]>
[2024-03-04T17:48:47.120+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T17:48:47.120+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 2
[2024-03-04T17:48:47.120+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-03-04T17:48:47.129+0000] {taskinstance.py:1300} INFO - Executing <Task(FileSensor): file_sensor_task> on 2024-03-04 17:48:44.892614+00:00
[2024-03-04T17:48:47.133+0000] {standard_task_runner.py:55} INFO - Started process 161 to run task
[2024-03-04T17:48:47.134+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'gera_dados_turbina', 'file_sensor_task', 'manual__2024-03-04T17:48:44.892614+00:00', '--job-id', '3397', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp_vq4atqt']
[2024-03-04T17:48:47.135+0000] {standard_task_runner.py:83} INFO - Job 3397: Subtask file_sensor_task
[2024-03-04T17:48:47.169+0000] {task_command.py:388} INFO - Running <TaskInstance: gera_dados_turbina.file_sensor_task manual__2024-03-04T17:48:44.892614+00:00 [running]> on host f002d2097047
[2024-03-04T17:48:47.208+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=adrianolima095@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=gera_dados_turbina
AIRFLOW_CTX_TASK_ID=file_sensor_task
AIRFLOW_CTX_EXECUTION_DATE=2024-03-04T17:48:44.892614+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-04T17:48:44.892614+00:00
[2024-03-04T17:48:47.213+0000] {base.py:73} INFO - Using connection ID 'fs_default' for task execution.
[2024-03-04T17:48:47.214+0000] {filesystem.py:64} INFO - Poking for file /home/adriano/ed/APP/data/data_turbine.json
[2024-03-04T17:48:47.214+0000] {filesystem.py:69} INFO - Found File /home/adriano/ed/APP/data/data_turbine.json last modified: 20240304174846
[2024-03-04T17:48:47.214+0000] {base.py:228} INFO - Success criteria met. Exiting.
[2024-03-04T17:48:47.218+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=gera_dados_turbina, task_id=file_sensor_task, execution_date=20240304T174844, start_date=20240304T174847, end_date=20240304T174847
[2024-03-04T17:48:47.266+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-03-04T17:48:47.294+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
